src/schemas.py

```
@@ -20,6 +21,171 @@
+        description="Configuration for dream functionality. If deriver is disabled, dreams will also be disabled and these settings will be ignored.",
+    )
+
+
+class SessionConfiguration(WorkspaceConfiguration):
+    """
+    The set of options that can be in a session DB-level configuration dictionary.
+
+    All fields are optional. Session-level configuration overrides workspace-level configuration, which overrides global configuration.
+    """
+
+    pass
```

Ruff (0.14.7)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)